bindings/python/src/lib.rs

```diff
@@ -39,37 +39,6 @@ pub use errors::*;
 mod options;
 pub use options::*;
 
-/// Apache OpenDAL™ Python binding
-///
-/// ## Installation
-///
-/// ```bash
-/// pip install opendal
-/// ```
-///
-/// ## Usage
-///
-/// ```python
-/// import opendal
-///
-/// op = opendal.Operator("fs", root="/tmp")
-/// op.write("test.txt", b"Hello World")
-/// print(op.read("test.txt"))
-/// print(op.stat("test.txt").content_length)
-/// ```
-///
-/// Or using the async API:
-///
-/// ```python
-/// import asyncio
-///
-/// async def main():
-/// op = opendal.AsyncOperator("fs", root="/tmp")
-/// await op.write("test.txt", b"Hello World")
-/// print(await op.read("test.txt"))
-///
-/// asyncio.run(main())
-/// ```
```

Comment:
Where do we need to place our docs?

Reply:
I just moved the quick start out from the modules

Other API docs and examples should be put in the rust binding files, using the form like below:

```rust
/// `AsyncOperator` is the entry for all public async APIs
///
/// Create a new `AsyncOperator` with the given `scheme` and options(`**kwargs`).
/// Args:
///     scheme(str): The service name that OpenDAL supports
///     **kwargs(Any): config passed to the operator
///
/// Example:
///     .. code-block:: python
///
///        import opendal
///        op = opendal.AsyncOperator("fs", path="/")
#[pyclass(module = "opendal")]
pub struct AsyncOperator {
    core: ocore::Operator,
    __scheme: ocore::Scheme,
    __map: HashMap<String, String>,
}
```